qemu/tests/ept_test.py

```diff
@@ -0,0 +1,40 @@
+from avocado.utils import process, cpu
+from virttest import env_process
+
+
+def run(test, params, env):
+    """
+    ept test:
+    1) Turn off ept on host
+    2) Check kvm_intel parameter
```

Comment:
could you make this more clear ? the purpose is check if read param cause host crash as we discussed.

Reply:
Updated.